mep3_simulation/launch/robot_launch.py

```diff
@@ -0,0 +1,84 @@
+import os
+import pathlib
+import launch
+from launch_ros.actions import Node
+from ament_index_python.packages import get_package_share_directory
+from webots_ros2_core.webots_launcher import WebotsLauncher
+
+
+def generate_launch_description():
+    package_dir = get_package_share_directory('mep3_simulation')
+    robot_description = pathlib.Path(os.path.join(package_dir, 'resource', 'webots_robot_description.urdf')).read_text()
+    ros2_control_params = os.path.join(package_dir, 'resource', 'ros2_control_configuration.yml')
+
+    # The WebotsLauncher is a Webots custom action that allows you to start a Webots simulation instance.
+    # It searches for the Webots installation in the path specified by the `WEBOTS_HOME` environment variable and default installation paths.
+    # The accepted arguments are:
+    # - `world` (str): Path to the world to launch.
+    # - `gui` (bool): Whether to display GUI or not.
+    # - `mode` (str): Can be `pause`, `realtime`, or `fast`.
+    webots = WebotsLauncher(
+        world=os.path.join(package_dir, 'webots_data', 'worlds', 'eurobot_2022.wbt')
+    )
+
+    sleep_time = 60 if 'CI' in os.environ else 10
+    controller_manager_timeout = ['--controller-manager-timeout', '50'] if os.name == 'nt' else []
+    controller_manager_prefix = 'python.exe' if os.name == 'nt' else f"bash -c 'sleep {sleep_time}; $0 $@' "
+
+    diffdrive_controller_spawner = Node(
+        package='controller_manager',
+        executable='spawner.py',
+        output='screen',
+        prefix=controller_manager_prefix,
+        arguments=['diffdrive_controller'] + controller_manager_timeout,
+    )
```

Comment:
```suggestion
    controller_manager_timeout = ['--controller-manager-timeout', '50']

    diffdrive_controller_spawner = Node(
        package='controller_manager',
        executable='spawner.py',
        output='screen',
        arguments=['diffdrive_controller', ] + controller_manager_timeout,
    )
```
Nemamo potrebe da podrzavamo Windows i `--controller-manager-timeout` je dostupan sada u Foxy